core/templates/dev/head/pages/collection_editor/collection_editor.html

```diff
@@ -97,6 +97,7 @@
   <script src="{{TEMPLATE_DIR_PREFIX}}/domain/collection/CollectionRightsBackendApiService.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/domain/collection/CollectionUpdateService.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/domain/collection/CollectionValidationService.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/domain/collection/SearchExplorationsBackendApiService.js"></script>
```

Comment:
The script list here is ordered alphabetically, so we'd want this tag above SkillListObjectFactory.